CONTRIBUTING.md

```diff
@@ -1,4 +1,92 @@
-# Contributing
+# Contributing to LumApps Extensions Templates
+
+<h2>Table of contents</h2>
+
+- [Contributing to LumApps Extensions Templates](#contributing-to-lumapps-extensions-templates)
+  - [Found an issue?](#found-an-issue)
+  - [Want a feature?](#want-a-feature)
+  - [Submission guidelines](#submission-guidelines)
+    - [Submitting an issue](#submitting-an-issue)
+    - [Submitting a pull request](#submitting-a-pull-request)
+  - [Coding rules](#coding-rules)
+  - [Git commit guidelines](#git-commit-guidelines)
+  - [Publish](#publish)
+
+## Found an issue?
+If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue to our [GitHub Repository](https://github.com/lumapps/lumapps-extensions-templates/issues).
+
+If you are feeling it, you can even fix the issue yourself and submit a Pull Request. Before opening a Pull Request, please see the Submission Guidelines below.
```

Comment:
`If you feel like it` would be better imo
`you can fix the issue and submit` (`yourself` is a bit too repetitive)